airflow/datadog_checks/airflow/airflow.py

```diff
@@ -86,6 +96,51 @@ def _submit_healthy_metrics_stable(self, resp, tags):
 
         self.gauge('airflow.healthy', int(health_status == AgentCheck.OK), tags=tags)
 
+    def _get_all_task_instances(self, url, tags):
+        resp = self._get_json(url)
+        if not resp:
+            return None
+
+        total_tasks = resp.get('total_entries')
+        self.log.info("Total running tasks: %s", total_tasks)
+        self.gauge('airflow.dag.task.total_running', total_tasks, tags=tags)
+
+        return resp.get('task_instances')
+
+    def _calculate_task_ongoing_duration(self, tasks, tags):
+        """
+        Calculate the ongoing duration of each task in the given list of tasks.
+        """
+        if not tasks:
+            self.log.info("No running tasks found")
+            return
```

Comment:
Isn't this a bit redundant if you're already checking for task_instances in line 56?